apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/entity/repository/WaUiMetadataRepository.java

```diff
@@ -46,17 +37,11 @@ Long countByPageAndQuestionIdentifier(
 
     @Query("Select COALESCE(Max(ui.orderNbr),0) FROM WaUiMetadata ui WHERE ui.waTemplateUid.id =:pageId")
     Integer findMaxOrderNbrForPage(@Param("pageId") Long pageId);
+    
+    public void deleteByWaTemplateUid(WaTemplate page);
+
 
-    public void deleteAllByWaTemplateUid(WaTemplate template);
```

Comment:
Another PR had changed `deleteByWaTemplateUid` to `deleteAllByWaTemplateUid`.  We want to keep `deleteAllByWaTemplateUid`